### DISCUSSION TITLE
Unable to reproduce output from demo

### DISCUSSION THREAD

**Discussion by Vagif12** [coqui-ai/TTS#1358] [General Q&A]
Hey there!

in the sample demo(https://erogol.github.io/ddc-samples/) the sentence:

> Bill got in the habit of asking himself “Is that thought true?” and if he wasn’t absolutely certain it was he just let it go.

has correctly pronounces the word` wasn't ` .

However, when running the command:

`tts --text "Bill got in the habit of asking himself “Is that thought true?” and if he wasn’t absolutely certain it was he just let it go."`

The intonation is different, and the word `wasn't` is pronounced incorrectly.

Do you have the scripts used to reproduce the samples of the demo? If not, what is the solution to this?
Thanks
Vagif

**Comment by erogol** [comment]
Try ```'``` instead of ```’``` 

And you need to increase the max_decoder_steps in the model config to convert all the text.

**Reply by Vagif12** [reply]
@erogol Yes I've increase the max_decoder_steps already, sorry forgot to mention that.

The command is now changed to:
`!tts --text "Bill got in the habit of asking himself \"Is that thought true?\" and if he wasn't absolutely certain it was he just let it go."`

But unfortunately it's still not working. 
Here is the link to the wav file: https://a.uguu.se/RWxaaCgN.wav

**Comment by Vagif12** [comment]
Perhaps this is an issue specific to Colab?
@erogol what do you think?

**Comment by Vagif12** [comment]
I've not managed to fix this after a day of tweaking. Could it be because of the fact that I upgraded numpy before running the command?

**Reply by robinsingh1** [reply]
++ wanted to follow up with this, experiencing the same issue
